src/SeqViz.tsx

```diff
@@ -42,6 +42,11 @@ export interface SeqVizProps {
   /** nucleotides keyed by symbol or index and the color to apply to it */
   bpColors?: { [key: number | string]: string };
 
+  children?: (props: CustomChildrenProps) => React.ReactNode;
+
+  /** Custom child circular viewer ref */
+  circularRef?: React.RefObject<HTMLElement>;
```

Comment:
This adds quite a few props. More when you add the genome viewer. What if we nested all the refs in a refs prop like:

```jsx
export interface SeqVizChildRefs {
  linear?: React.RefObject<HTMLElement>;
  circular?: React.RefObject<HTMLElement>;
}

/** `SeqViz` props. See the README for more details. One of `seq`, `file` or `accession` is required. */
export interface SeqVizProps {

  refs?: SeqVizChildRefs;
```

Reply:
I think that should work, let me change it and update the PR